goals/Oct-1.md

```diff
@@ -0,0 +1,24 @@
+## Goals for Oct. 3 - 14
+
+### Experiments
+**Week 1**
+- **Oct 3**: Luminescence testing with cpec library polymerase assay. Send for sequencing mutants that produce statistically distinct results.
+- **Oct 4**: Learn proper tc technique from Kim.
+- **Oct 5**: I have an interview in the afternoon, so probably can't come into lab.
+- **Oct 6**: Purify and send for sequencing hacky-taq PCR products from mutagenesis with [Mn] of 1.5, 2.0, 2.5, and 3.0 (mM).
+Justification: Preliminary results from comparing [Mn] of 0, 2.5, and 5.5 indicate that more mutations occurred with 2.5mM than with 5.5mM (0mM: 1, 2.5mM: 8, 5.5mM: 3)
+- **Oct 7**: Analyze results from cpec sequencing.
+
+**Week 2**
+- Successfully produce mutated PCR products of PB2 part 1, either with Agilent as before (try cutting it into even smaller fragments?) or with hacky-taq, if one of the concentrations reliably induces a respectable number of mutations.
+
+
+### Computational
+**Week 1**
+- simulate coin_flip and tcm bayesian analysis with thinkbayes methods + Chapter 2 of Think Bayes
+- still need to figure out Biopython...seem to be having trouble with reading files

```

Comment:
Also, if you're stuck on a programming question for more than a few hours of banging your head on it, just ask :smile:. I think it's still a higher priority to make strides solving the bigger problems.